packages/x-adapter-platform/src/mappers/__tests__/response/search-response.mapper.spec.ts

```diff
@@ -0,0 +1,385 @@
+import { searchResponseMapper } from '../../response/search-response.mapper';
+import { PlatformSearchResponse, SearchResponse } from '../../../types/response.types';
+import { platformSearchResponse } from '../../../__tests__/__fixtures__/platform-search.response';
+
+describe('search response mapper tests', () => {
+  it('should map the response', () => {
+    const response: SearchResponse = {
+      banners: [
```

Comment:
Maybe we can move these responses to a file and mix the skuSearch and the search responses:

const skuSearch = {
  ....
  ....
  }
  
  const searchResponse = {
     banners: [],
     ...the rest of properties that doesnt exist on the skuSearch
     ...skuSearch
  }
  
  Im not sure if this is better and clean, mix both responses